src/cli/keys.js

```diff
@@ -73,6 +76,9 @@ module.exports = ({ commandProcessor, root }) => {
 				number: true,
 				description: 'Port number of the server to add to the key'
 			},
+			'device': {
+				description: 'Device ID of the target device',
+			},
```

Comment:
we have a number of commands in this same namespace that accept `deviceID` as a _parameter_ - e.g. https://github.com/particle-iot/particle-cli/blob/5659edf2001163fa4f88a3db6057c91276e74e9a/src/cli/keys.js#L56-L63 

i think we should either support that call signature (total pain, requires manual arg checking / juggling, etc) - OR - we rename the option (flag) to `--deviceID`. in both cases we want to be more explicit about the expected input (since `device` means `id` or `name` across the rest of the CLI commands)